docs/doc_sources/beginners_guides/device_info.rst

```diff
@@ -0,0 +1,59 @@
+.. _beginners_guide_device_info:
+
+Obtaining information about device
+==================================
+
+An instance of :py:class:`SyclDevice` provides access to a collection of information
```

Comment:
while :py:class:`SyclDevice` is a hyperlink in other places here it is not.

Reply:
should be fixed